custom_metrics/almanac.js

```diff
@@ -12,32 +12,263 @@
 // 3. Test your change by following the instructions at https://github.com/HTTPArchive/almanac.httparchive.org/issues/33#issuecomment-502288773.
 // 4. Submit a PR to update this file.
 
-return {
+return JSON.stringify({
   // Wether the page contains <script type=module>.
   '01.12': document.querySelector('script[type=module]') ? 1 : 0,
   // Wether the page contains <script nomodule>.
   '01.13': document.querySelector('script[nomodule]') ? 1 : 0,
   'link-nodes': (() => {
     // Returns a JSON array of link nodes and their key/value attributes.
-    // Used by 01.14, 01.15, 01.16...
+    // Used by 01.14, 01.15, 01.16, 10.6,  06.46, 12.18
+    function parseNodes(nodes) {
```

Comment:
Are you able to pull the function out of the return block so it can be called by multiple metrics? We don't want to define the same function twice because we're still duplicating code.